.github/workflows/codebase-review.yml

```
@@ -7,38 +7,64 @@ jobs:
+          CROFAI_API_KEY: ${{ secrets.CROFAI_API_KEY }}
         run: |
-          DEFAULT_BRANCH="${{ github.event.repository.default_branch }}"
+          FILES=$(find . -type f \( -name "*.py" -o -name "*.rs" -o -name "*.ts" -o -name "*.svelte" -o -name "*.js" -o -name "*.tsx" -o -name "*.jsx" -o -name "*.go" -o -name "*.rb" -o -name "*.java" -o -name "*.c" -o -name "*.h" -o -name "*.toml" -o -name "*.yaml" -o -name "*.yml" -o -name "*.json" -o -name "*.md" \) ! -path "*/node_modules/*" ! -path "*/target/*" ! -path "*/.git/*" ! -path "*/dist/*" ! -path "*/.venv/*" ! -path "*/__pycache__/*" ! -path "*/vendor/*" | head -100)
 
-          if ! command -v pr-agent &>/dev/null; then
-            echo "ERROR: pr-agent not installed globally. Run: uv tool install pr-agent --with tiktoken --python 3.12"
-            exit 1
-          fi
+          PAYLOAD=""
+          PAYLOAD="$PAYLOAD# Repository: ${{ github.repository }}
+"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)